dask/dataframe/tests/test_multi.py

```diff
@@ -247,14 +249,25 @@ def test_indexed_concat(join):
                      index=[1, 2, 4, 5, 6, 8])
     b = dd.repartition(B, [1, 2, 5, 8])
 
-    result = concat_indexed_dataframes([a, b], join=join)
-    expected = pd.concat([A, B], axis=0, join=join)
+    with warnings.catch_warnings(record=True) as w:
```

Comment:
The hope is that these tests are future-proof when pandas switches the default to `sort=False`. Then pandas should no longer emit warnings, but the tests will still pass (and we'll be free to clean this up).